beehive.go

```diff
@@ -50,10 +50,9 @@ var (
 func main() {
 	app.AddFlags([]app.CliFlag{
 		{
-			V:     &configFile,
-			Name:  "config",
-			Value: "./beehive.conf",
```

Comment:
The nice thing about keeping the default value defined here, is that it actually shows up as a helpful little reminder when running `beehive --help`:

```
  -config string
        Config-file to use (default "./beehive.conf")
```

Reply:
So the issue I had with this, is that with the default changed to OS specific locations, that help in string is a bit longer and not very pretty. It's only a cosmetic/readability issue though, I'm happy to set the default value to the new/preferred OS dependent location. Whatcha think?

Reply:
Done :heavy_check_mark:, doesn't look that bad.